File: CGViewRecorder/TC/ViewController.m
//
//  ViewController.m
//  TC
//
//  Created by vector on 2019/11/15.
//  Copyright © 2019 vector. All rights reserved.
//

#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view.
    
    
    void * p = malloc(UINT32_MAX);
    //success
    NSLog(@"p", p);
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}




@end




typedef uint64_t CCIndex;
typedef uint8_t * CCPagePtr;

File: Common/CC/CC/ViewController.swift
//
//  ViewController.swift
//  CC
//
//  Created by vector on 2020/1/27.
//  Copyright © 2020 vector. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        
        OCBridge.testAes()
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

